game/game.py

```diff
@@ -514,6 +514,7 @@ def compute_unculled_zones(self) -> None:
             zones.append(package.target.position)
 
         self.__culling_zones = zones
+        events.update_unculled_zones()
```

Comment:
Add missing event-logic to `compute_unculled_zones`, justifies the other changes in the file...